go-controller/pkg/clustermanager/egressip_controller_test.go

```diff
@@ -2190,6 +2191,50 @@ var _ = ginkgo.Describe("OVN cluster-manager EgressIP Operations", func() {
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		})
 
+		ginkgo.It("ensure egressIP status is in sync with cloud private ip config", func() {
```

Comment:
++ THANK YOU for the tests, we are really missing coverage for UTs around non BM platform types..